public String divide(Integer a, Integer b){
    //code here
    try {
        
        return String.valueOf(a/b);       
    }catch(ArithmaticException e) {
        System.debug('The following exception has occurred: ' + e.getMessage());
    }
    
}